docs/user/managing-devices.md

```diff
@@ -0,0 +1,266 @@
+# Managing Devices
+
+## Enrolling Devices
+
+The first time the Flight Control Agent runs, it generates a cryptographic key pair that serves as the device's unique cryptographic identity. The pair's private key never leaves the device, so that the device cannot be duplicated or impersonated. The cryptographic identity is registered with the Flight Control Service during enrollment and gets wiped during device decommissioning.
+
+When the device is not yet enrolled, the agent performs service discovery to find its Flight Control Service instance. It then establishes an secure, mTLS-protected network connection to the Service using the X.509 enrollment certificate is has been provided during image building or device provisioning. Next, it submits an Enrollment Request to the service that includes a description of the device's hardware and operating system as well as an X.509 Certificate Signing Request (CSR) including its cryptographic identity to obtain its initial agent certificate. At this point, the device is not yet considered trusted and therefore remains quarantined in a "waiting room" until its Enrollment Request has been approved or denied by an authorized user (e.g. a administrator, an installer persona, or an auto-approver process).
+
+### Enrolling using the Web UI
+
+### Enrolling using the CLI
+
+You can list all devices currently in the waiting room by running the following command:
```

Comment:
Probably not, no. Any suggestion how to call this? AWS calls it the "device lobby", for example.

Reply:
Maybe something more explicit? In this context, it could just be "You can list all devices currently waiting to be approved / pending approval...".

Otherwise maybe "pending devices lobby"?

Reply:
Makes sense, thanks. Updated.